plugin-locationcomponent/src/test/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImplTest.kt

```diff
@@ -39,7 +41,8 @@ class LocationComponentPluginImplTest {
   private val typedArray = mockk<TypedArray>(relaxed = true)
   private val pack = "com.mapbox.maps"
   private val drawable = mockk<Drawable>(relaxed = true)
-
+  private val windowManager = mockk<WindowManager>(relaxed = true)
+  private val sensorManager = mockk<SensorManager>(relaxed = true)
```

Comment:
perhaps use `relaxedUnitFun = true` to make sure we don't mock too much

Reply:
This would cause `io.mockk.MockKException: no answer found for: SensorManager(#238).getDefaultSensor(11)`